Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{etc}/fpc.cfg"
   end
 
   test do
     hello = <<-EOS.undent
       program Hello;
+      uses GL;
```

Comment:
No, sorry, `#{prefix}/etc` is the wrong location, we'll need to fix this up so the configuration file is written into `/usr/local/etc` as otherwise it'll be destroyed on reinstall and upgrades (which sucks for configuration files).